nuxeo-services/nuxeo-platform-types-core/src/test/java/org/nuxeo/ecm/platform/types/SubtypesJsonEnricherTest.java

```diff
@@ -95,4 +108,20 @@ public void test() throws Exception {
         json.childrenContains("type", "DummyDoc");
     }
 
-}
\ No newline at end of file
+    @Test
+    public void testSubtypesWithLocalConfiguration() throws Exception {
```

Comment:
Worth adding more tests to validate the whole enricher:
- denyAllTypes
- allowedTypes
- deniedTypes
- mix of properties